app/views/requests/index.html.erb

```diff
@@ -1,5 +1,3 @@
-<p id="notice"><%= notice %></p>
```

Comment:
Why is the notice removed here?
This change seems to remove value without a reason for it, especially as you are still adding a notice in the requests controller.

Reply:
I removed it because I received double notices when I was falsely redirecting to this page. I will add it again.